### DISCUSSION TITLE
Conditional blocks based on screen size

### DISCUSSION THREAD

**Discussion by shadigaafar** [WordPress/gutenberg#40564] [Ideas]
It would be nice if you can display blocks based on screen size.  sometimes you want to have totally different in design block in small screens. there should be a setting to choose the display screen size of the block. and to make it even easier instead of adding this setting support to all blocks, allow this feature only in the group block. so, if you want to control the display of any block, you just have to insert it into a group block then choose the screen size.

**Comment by tnchuntic** [comment]
Also it would be nice if Templates / Blocks can be displayed/hide base on page/post.
